libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigation.java

```diff
@@ -1022,7 +1023,8 @@ private void initializeForTest(Context context) {
    * to prevent users from removing it.
    */
   private void initialize() {
-    mapboxNavigator = new MapboxNavigator(new Navigator());
+    Navigator navigator = configureNavigator();
```

Comment:
unnecessary change

Reply:
What do you mean by unnecessary change?

Something like this? 

```suggestion
    mapboxNavigator = new MapboxNavigator(configureNavigator());
```

Reply:
I pushed ☝️ could you confirm that's what you meant when you have a chance @RingerJK?

Reply:
it's just nit :) now looks better